src/ui/pages/IdentifierDetails/IdentifierDetails.test.tsx

```diff
@@ -513,7 +513,7 @@ describe("Individual Identifier details page", () => {
     });
 
     act(() => {
-      fireEvent.click(getByTestId("rotate-key-button"));
+      fireEvent.click(getByTestId("primary-button-rotate-key"));
     });
```

Comment:
Please add something to check for all the text in the page.
![07112024122132](https://github.com/user-attachments/assets/c35d9fb4-9408-40e7-b310-6b3e462a801a)